app/assets/javascripts/map.js

```diff
@@ -2,10 +2,26 @@ $(function() {
   function initialize() {
     var mapOptions = {
       center: { lat: centerLocation.latitude, lng: centerLocation.longitude },
-      zoom: 13
+      zoom: 12
     };
     var map = new google.maps.Map(document.getElementById('map-canvas'),
         mapOptions);
+
+    sitterMark = sitterLocation.forEach(function(sitter) {
+      var marker = new google.maps.Marker({
+          position: new google.maps.LatLng(sitter.latitude, sitter.longitude),
+          map: map
+      });
+    });
+
+    $("#search-sitter-container").on("click", ".sitter-profile", function() {
+      map = new google.maps.Map(document.getElementById('map-canvas'));
+      var laLatLng = new google.maps.LatLng(sitterLocation[0].latitude, sitterLocation[0].longitude);
```

Comment:
Line is too long.<br>'google' is not defined.<br>'sitterLocation' is not defined.